README.md

```diff
@@ -2,10 +2,11 @@
 
 ![screenshot](papers/screenshot1.png)
 
-a multi-system chiptune tracker.
+Furnace, a multi-system chiptune tracker.
 
-[downloads](#downloads) | [discussion](#discussion) | [help](#help) | [developer info](#developer-info)
+Jump to a section: [downloads](#downloads) | [discussion](#discussion) | [help](#help) | [developer info](#developer-info) | [unofficial packages](#unofficial-packages) | [faq](#faq)
```

Comment:
Not sure whether it's actually necessary to add "Jump to a section". It's meant to look like a menu bar.

Reply:
I've removed the "Jump to a section part"

Reply:
Looks good, but I feel like a poll might be necessary.

Reply:
Maybe. Perhaps you could make a poll on the Discord server?